Makefile

```diff
@@ -1,4 +1,4 @@
-DOCKER_COMPOSE := docker-compose
+DOCKER_COMPOSE := docker compose
```

Comment:
docker-compose is now built into docker, so this this gets rid of this warning:

```
Docker Compose is now in the Docker CLI, try `docker compose up`
```

Reply:
nit: This change seems to break compatibility with older version of Docker where `docker compose` is not built-in. Maybe it's something we could provide?